charts/aws-ebs-csi-driver/templates/_node.tpl

```diff
@@ -8,6 +8,10 @@ metadata:
   namespace: {{ .Release.Namespace }}
   labels:
     {{- include "aws-ebs-csi-driver.labels" . | nindent 4 }}
+  {{- with .Values.node.daemonSetAnnotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
```

Comment:
Indentation is inconsistent with the rest of this file, suggest:

```
  annotations:
    {{- with .Values.node.daemonsetAnnotations }}
    {{- toYaml . | nindent 4 }}
    {{- end }}
```

Reply:
Counterpoint: https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/489be46f0252248e8534f636e049fff98ac380ca/charts/aws-ebs-csi-driver/templates/_node.tpl#L29-L32

We can have a followup PR that makes this consistent across all templates.